src/DI/Container.php

```diff
@@ -98,8 +98,10 @@ public function removeService($name)
 	 */
 	public function getService($name)
 	{
-		$name = isset($this->meta[self::ALIASES][$name]) ? $this->meta[self::ALIASES][$name] : $name;
 		if (!isset($this->registry[$name])) {
+			if (isset($this->meta[self::ALIASES][$name])) {
+				return $this->getService($this->meta[self::ALIASES][$name]);
+			}
```

Comment:
That I prefer and propose the version if extra `getService` call. It is a simple change which improve `getService` performance in most-common use-case (no aliases).